### PR TITLE
Convert to swift3

### PR DESCRIPTION
**Pull request by JamesDurham**
Conversion done
fixed some title comment strings:
   filename.swift in comment jive now with filenames

### CODE REVIEW

File: RealmModelGenerator.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -605,6 +605,7 @@
 				ONLY_ACTIVE_ARCH = YES;
 				SDKROOT = macosx;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				SWIFT_VERSION = 3.0;
```
Comment (berbschloe) [inline RealmModelGenerator.xcodeproj/project.pbxproj:608]: Swift 4.0 is the newest version. Lets migrate to that.

File: RealmModelGenerator/NibDesignable.swift [hunk 2]
```diff
@@ -68,18 +69,19 @@ extension NSView {
      - returns: Name of a single view nib file.
      */
     public func nibName() -> String {
-        return self.dynamicType.description().componentsSeparatedByString(".").last!
+        return String(type(of: self).description().split(separator: ".").last!)//.componentsSeparatedByString(".").last!
```
Comment (berbschloe) [inline RealmModelGenerator/NibDesignable.swift]: go ahead and remove comment

File: dummy.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+Dummy
```
Comment (berbschloe) [inline dummy.txt]: Delete dummy.txt

**Review by berbschloe — CHANGES_REQUESTED** [review]
Just three small changes. Thanks

File: RealmModelGenerator.xcodeproj/project.pbxproj (hunk 1 above)
Comment (JamesDurham) [reply RealmModelGenerator.xcodeproj/project.pbxproj:608]: That's the next pull request not this one.

File: RealmModelGenerator/NibDesignable.swift (hunk 2 above)
Comment (JamesDurham) [reply RealmModelGenerator/NibDesignable.swift]: How do I make this modification. 
Do I add another commit or what?

File: dummy.txt (hunk 3 above)
Comment (JamesDurham) [reply dummy.txt]: How do I do this also

File: RealmModelGenerator/NibDesignable.swift (hunk 2 above)
Comment (berbschloe) [reply RealmModelGenerator/NibDesignable.swift]: you would make another commit saying that you removed a comment

File: dummy.txt (hunk 3 above)
Comment (berbschloe) [reply dummy.txt]: You added the dummy.txt as apart of this pull request. Just go ahead and make another commit deleting it.

File: RealmModelGenerator.xcodeproj/project.pbxproj (hunk 1 above)
Comment (berbschloe) [reply RealmModelGenerator.xcodeproj/project.pbxproj:608]: 👍

**Comment by berbschloe** [comment]
B4 you merge this into master. Squash all your changes into one commit so the integration point into master is clean. Amend the commit message to say something like "Migrate to Swift 3" and all of the changes are one atomic unit. There is no reason to preserve git history. This is the normal pattern we do for all of our projects.

**Pull request merged**